From 1d474c29c8708655ca6444bcfd04180965cb4c77 Mon Sep 17 00:00:00 2001
From: Conor Patrick <conorpp94@gmail.com>
Date: Thu, 3 Sep 2020 12:19:46 -0700
Subject: [PATCH] pin cortex-m-semihosting for logging

---
 components/logging/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/logging/Cargo.toml b/components/logging/Cargo.toml
index 1ed8daf..9b48aaf 100644
--- a/components/logging/Cargo.toml
+++ b/components/logging/Cargo.toml
@@ -9,7 +9,7 @@ edition = "2018"
 [dependencies]
 cfg-if = "0.1.10"
 cortex-m-funnel = { git = "https://github.com/conorpp/cortex-m-funnel"}
-cortex-m-semihosting = { git = "https://github.com/rust-embedded/cortex-m-semihosting"}
+cortex-m-semihosting = { git = "https://github.com/rust-embedded/cortex-m-semihosting#442fb883"}
 nb = "1"
 ufmt = {version =  "0.1.0"}
 
-- 
2.28.0.windows.1

